modules/openapi-generator/src/main/resources/rust-axum/models.mustache

```diff
@@ -207,7 +207,6 @@ use crate::{models, types::*};
     #[cfg_attr(feature = "conversion", derive(frunk::LabelledGeneric))] 
     pub struct {{{operationIdCamelCase}}}QueryParams {
         {{#queryParams}}
-          {{#vendorExtensions}}
```

Comment:
@tanadeau could you please explain why we need to remove `vendorExtensions`?

Reply:
This created the wrong context, and within it, the template couldn't get the correct values for `isArray`, etc. It's now also consistent with the other blocks for header params, path params, etc. They would have had the same issue.